packages/trader/src/Modules/CFD/Containers/cfd-password-manager-modal.tsx

```diff
@@ -155,12 +170,13 @@ const InvestorPasswordManager = ({
     setPasswordType,
     toggleModal,
     validatePassword,
-}) => {
+}: TInvestorPasswordManager) => {
     if (is_submit_success_investor) {
         return <CFDPasswordSuccessMessage toggleModal={toggleModal} is_investor />;
     }
 
-    const initial_values = { old_password: '', new_password: '', password_type: 'investor' };
+    const typeInvestor = 'investor';
```

Comment:
```suggestion
    const type_investor = 'investor';
```
should be in camel case as per our code standards - https://github.com/binary-com/deriv-app/blob/master/docs/JavaScript/README.md#naming-conventions-variables